app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide(:object, params[:object]) %>
```

Comment:
you don't need this file and render errors in form partial
bug with flash error show was fixed - please rebase with the latest code from master